src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -110,14 +172,22 @@ class TableView extends React.Component<TableViewProps> {
    * Please read the comment on `_createColumn`
    */
   _deleteColumn = (columnIndex: number) => {
-    const {location, eventView, tableData} = this.props;
+    const {location, eventView, tableData, organization} = this.props;
 
     if (!tableData || !tableData.meta) {
       return;
     }
 
     const nextEventView = eventView.withDeletedColumn(columnIndex, tableData.meta);
 
+    // metrics
+    trackAnalyticsEvent({
+      eventKey: 'discover_v2.delete_column',
+      eventName: 'Discoverv2: A column was deleted',
+      deleted_at_index: columnIndex,
+      organization_id: organization.id,
+    });
```

Comment:
It'll be useful to track which column key is being deleted. It can provide valuable info to optimize our default queries.

Reply:
I've confirmed with Bowen on this; he's fine with not having to track this.

Reply:
I've added `aggregation`, `field`, and `fieldname`.